frontend/packages/dev-console/src/components/pipelines/pipelinesResources/pipelinesResource-utils.ts

```diff
@@ -1,20 +1,36 @@
 import * as _ from 'lodash';
 import { k8sCreate, K8sResourceKind } from '@console/internal/module/k8s';
+import { SecretModel } from '@console/internal/models';
 import { PipelineResourceModel } from '../../../models';
 
-// TODO add interface amd check for values
+export interface ParamData {
+  [key: string]: any;
+}
 
 const getRandomChars = (digit = 6): string => {
   return Math.random()
     .toString(36)
     .replace(/[^a-z0-9]+/g, '')
     .substr(1, digit);
 };
+
+const getDefinedObj = (objData: ParamData): ParamData => {
+  const processedData = {};
+  _.each(objData, (value, name) => {
+    if (value) {
+      processedData[name] = value;
+    }
+  });
+  return processedData;
+};
+
```

Comment:
use lodash function to skip undefined and null values
```suggestion
return _.omitBy(objData, v => _.isUndefined(v) || _.isNull(v))
```